lib/db/mysql.js

```diff
@@ -1156,7 +1156,7 @@ module.exports = function (log, error) {
         }
 
         var mode = REQUIRED_SQL_MODES.join(',')
-        connection.query('SET SESSION sql_mode = \'' + mode + '\'', function(err) {
+        connection.query(`SET SESSION sql_mode = ${mode};`, (err) => {
```

Comment:
you *might* need the `' '` around ${mode} per example in https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html

`SET SESSION sql_mode = 'modes';`

Reply:
Yeah i changed it after this, need to squash commits :(